src/lib/y2ntp_client/yast-timesync.timer.erb

```diff
@@ -0,0 +1,7 @@
+[Timer]
```

Comment:
I'm just not sure if `lib` subdir is the best place for this, it's a data file, you cannot `require` it in Ruby.

In registration I use [`data/registration`](https://github.com/yast/yast-registration/tree/master/src/data/registration) subdirectory (which is installed into `/usr/share/YaST2/data/registration/`).